src/internal/views/nunjucks/gauging-stations/licences.njk

```diff
@@ -0,0 +1,223 @@
+            value: {
+              text: [data.stations[data.stationID].stationReference],
+              classes: "meta-data__value"
+            }
+          } if data.stations[data.stationID].stationReference
+        ]
+        }) }}
+      {% endset %}
+
+    {{ govukDetails({
+      summaryText: "Station details",
+      html: metaData
+    }) }}
+
+  </div>
+</div>
+
+
+{% set tags = data.stations[data.stationID]['tags'] %}
+{% if tags.length %}
+  <div class="govuk-grid-row govuk-!-margin-bottom-6">
+    <div class="govuk-grid-column-full">
+      {{ govukButton({
+    text: "Create a water abstraction alert",
+    href: "licence-notices/send-a-water-abstraction-alert/select-the-type-of-alert?stationID=" + [data.stationID]
+  }) }}
+    </div>
+  </div>
+{% endif %}
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+
+    <!-- Start tag-table.html: Licences linked to this monitoring station -->
+    {% block service %}
+        <!--setting the loop number to 0-->
+        {% set loopNumber = "" %}
+
+        <!--getting the licence data-->
+        {% set tags = data.stations[data.stationID]['tags'] %}
+        {% if tags.length %}
+          <table class="govuk-table">
+            <caption class="govuk-table__caption govuk-table__caption--m">Licences linked to this monitoring station</caption>
+            <thead class="govuk-table__head">
+              <tr class="govuk-table__row">
+                <th scope="col" class="govuk-table__header ">Licence</th>
+                <th scope="col" class="govuk-table__header ">Abstraction period</th>
+                <th scope="col" colspan="3" class="govuk-table__header ">Flow and level restriction<br> type and threshold </th>
+                <th scope="col" colspan="2" class="govuk-table__header ">Last type of alert  <br>sent and date issued</th>
+              </tr>
+            </thead>
+            <tbody class="govuk-table__body">
+            <!--starting to loop through the bill runs data -->
+
+            {% for i in tags %}
+              <!--setting the loop number -->
+              {% set loopNumber = loop.index0 %}
+              <tr class="govuk-table__row">
+                <td class="govuk-table__cell">
+                  <!--set the link to go to a specific licence-->
+                  {% for licence in data['licences']%}
+                    {% if licence.number == i.licenceNumber %}
+                      <a href="licence?ID={{loop.index0}}" class="govuk-link">
+                    {% endif %}
+                  {% endfor %}
+                  {{i.licenceNumber}}</a>
+                </td>
+
+                <!--get the tags for that individual licences-->
+                {% set licenceTags = i['tagValues'] %}
+
+                <td class="govuk-table__cell ">
+                  {% set period = licenceTags[0].abstractionPeriod %}
+                  {{ period.periodStart | abstractionPeriod }} to {{ period.periodEnd | abstractionPeriod }} <br>
```

Comment:
As mentioned above this could be shortened to:
```
{{ tag.abstractionPeriod | abstractionPeriod }}
```
if the service was responding with the standardised shape.